ios/Classes/Blemulator/SimulatorPlugin.h

```diff
@@ -0,0 +1,7 @@
+#import <Flutter/Flutter.h>
+
+@interface SimulatorPlugin : NSObject<FlutterPlugin>
+
+- (instancetype)initWithPlatformToDartChannel:(FlutterMethodChannel *)platformToDartChannel;
```

Comment:
Is it necessary?

Reply:
It is not 'necessary' but this conformance automatically adds `+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar` which is required in the solution.

Other solution would be to just define similar method in the class and just specify `NSObject`  as a base class.